### DISCUSSION TITLE
Can casting be supported, or redirect to the YouTube app for casting purposes?

### DISCUSSION THREAD

**Discussion by fahad86** [TeamNewPipe/NewPipe#10869] [Questions] [answered]
It would be very good if there would be a solution for casting currently playing videos to a Google casting device (e.g. Google TV)

**Comment by opusforlife2** [comment]
What happens if you use the cast button?

**Reply by fahad86** [reply]
thinking about it, it doesn't need to be a cast button.. need a way to open the current video in (deeplink into) the Youtube app...

Probably a button like: `Open in Youtube App`

I have currently set all youtube url's to open by default in NewPipe

In another note, I just noticed that the team had been working on this: https://github.com/TeamNewPipe/PipeCast .... Wondering what was going on with this and if and when it would be integrated with NewPipe

**Reply by opusforlife2** [reply]
> What happens if you use the cast button?

ಠ_ಠ

**Reply by fahad86** [reply]
https://youtu.be/r008q2jm0bw

**Comment by vejobo9414larland** [comment, marked as answer]
1. upper right corner within the playing video area, you need to click on the down arrow image (like “v”)
2. click “share” (image of three dots connected by two straight lines)
3.1. "send to" select the button with the image of the TV (next to the printer button), find your TV
or
3.2. select a system (or third-party) application for casting and use it. for example I have "web media share"

on TV try the application smarttube - open source, project on Github
